src/crimson/osd/object_context_list.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
how is the change related? (no criticism - just trying to understand...)

Reply:
thanks for pointing this out! this change should not be included in this PR.